src/extensions/default/Inline3DParametersEditor/ParameterEditor.js

```diff
@@ -0,0 +1,149 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var Strings            = brackets.getModule("strings"),
+        Mustache           = brackets.getModule("thirdparty/mustache/mustache");
+
+    /** Mustache template that forms the bare DOM structure of the UI */
+    var Template = require("text!ParameterEditorTemplate.html");
+
+     //used to control the smoothness of scroll.
+     var MODIFIER = 10;
```

Comment:
I have shifted the logic of the Modifier to the encapsulated JS module [here](https://github.com/mozilla/brackets/pull/651/files#diff-b2fd7b1c904a3a8965456f205f1ba5d3R24).

And a variable _modifier is included in the main file [here](https://github.com/mozilla/brackets/pull/651/files#diff-12ec682d75836bdc488d42abbe212a30R16).

Do I still need to modify this to _modifier_px ?